src/modules/design/property-panel/indextabs/general/sections/xor-gateway/xor-gateway.html

```diff
@@ -0,0 +1,10 @@
+<template>
+  <require from="./xor-gateway.css"></require>
+  <require from="../../../../styles/sections.css"></require>
+  <div class="section-panel">
+    <div class="panel__heading">Exclusive Gateway</div>
+    <div class="panel__content">
+      <p class="information-text">You can configure the conditions of each outgoing sequence flow of this gateway by clicking on the respective sequence flow.</p>
```

Comment:
```suggestion
      <p class="information-text">You can configure the conditions of each outgoing sequence flow, by clicking on the respective sequence flow.</p>
```

Liest sich flüssiger.